mirdata/datasets/compmusic_jingju_acappella.py

```diff
@@ -0,0 +1,399 @@
+    accompaniment and mixed versions.
+
+    Additional details:
+    Annotation format, units, parsing code and other information please refer to:
+    https://github.com/MTG/jingjuPhonemeAnnotation
+
+    License information:
+    Textgrid annotations are licensed under Creative Commons Attribution-NonCommercial 4.0 International License.
+    Wav audio ending with ‘upf’ or ‘lon’ is licensed under Creative Commons Attribution-NonCommercial 4.0 International.
+    For the license of .wav audio ending with ‘qm’ from C4DM Queen Mary University of London, please refer to
+    this page http://isophonics.org/SingingVoiceDataset
+
+"""
+
+import csv
+import os
+
+import numpy as np
+import librosa
+from mirdata import annotations, core, download_utils, io, jams_utils
+from typing import BinaryIO, Optional, TextIO, Tuple
+
+BIBTEX = """
+@dataset{rong_gong_2018_1323561,
+  author       = {Rong Gong and
+                  Rafael Caro Repetto and
+                  Yile Yang and
+                  Xavier Serra},
+  title        = {Jingju a cappella singing dataset part1},
+  month        = jul,
+  year         = 2018,
+  publisher    = {Zenodo},
+  version      = 7,
+  doi          = {10.5281/zenodo.1323561},
+  url          = {https://doi.org/10.5281/zenodo.1323561}
+}
+@article{black2014automatic,
+  title={Automatic identification of emotional cues in Chinese opera singing},
+  author={Black, Dawn AA and Li, Ma and Tian, Mi},
+  journal={ICMPC, Seoul, South Korea},
+  year={2014}
+}
+"""
+
+REMOTES = {
+    "annotation_txt": download_utils.RemoteFileMetadata(
+        filename="annotation_txt.zip",
+        url="https://zenodo.org/record/1323561/files/annotation_txt.zip?download=1",
+        checksum="851c9c3fe195fd20bec42d32ddd9deb7",
+        destination_dir=".",
+    ),
+    "catalogue_dan": download_utils.RemoteFileMetadata(
+        filename="catalogue - dan.csv",
+        url="https://zenodo.org/record/1323561/files/catalogue%20-%20dan.csv?download=1",
+        checksum="82ce90bd8508b1ae12c6a1fe489618a4",
+        destination_dir=".",
+    ),
+    "catalogue_laosheng": download_utils.RemoteFileMetadata(
+        filename="catalogue - laosheng.csv",
+        url="https://zenodo.org/record/1323561/files/catalogue%20-%20laosheng.csv?download=1",
+        checksum="768fa00ce1f8880ae5480fae103ecc06",
+        destination_dir=".",
+    ),
+    "readme": download_utils.RemoteFileMetadata(
+        filename="readme.txt",
+        url="https://zenodo.org/record/1323561/files/readme.txt?download=1",
+        checksum="f1113d4c03b379a6a23d85e2c215d54b",
+        destination_dir=".",
+    ),
+    "wav": download_utils.RemoteFileMetadata(
+        filename="wav.zip",
+        url="https://zenodo.org/record/1323561/files/wav.zip?download=1",
+        checksum="4722abda831c20b169a62b2754b15bea",
+        destination_dir=".",
+    ),
+}
+
+LICENSE_INFO = (
+    "upf or lon: Creative Commons Attribution Non-Commercial 4.0 International, "
```

Comment:
Maybe small comment such as "audio files ending with .." as you have in the description will be clearer